Testscripts/Linux/fpga-card-validation.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+########################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+########################################################################
+########################################################################
+#
+# Description:
+#   This script assumes Xilinx XRT tools are available on the VM
+#   and tries to run the validation test to make sure FPGA cards
+#   are functional. 
+#
+#
+########################################################################
+
+#######################################################################
+function prepare() {
+    xrt_setup="/opt/xilinx/xrt/setup.sh"
+    if [[ ! -f $xrt_setup ]]; then
+        LogErr "$xrt_setup file not found on VM!"
+        SetTestStateAborted
+        return 1
+    fi
+    source $xrt_setup
+}
+function validate_cards() {
+    LogMsg "Validating FPGA cards"
+    if ! [ -x "$(command -v xbutil)" ]; then
+        LogErr "xbutil not found in the path!"
+        SetTestStateAborted
+        return 1
+    fi
+    xbutil validate >> TestExecution.log
```

Comment:
xbutil is used, then it should be checked before it is available in the system,. Like other tests, any tool availability can be checked out before running.

Reply:
Could you please point me to a test so I can check? I am not clear on "it should be checked before it is available in the system". `xbutil` also comes from the XRT package. If the VM is deployed using a Canonical Ubuntu, for example, these tools will not be available.

Reply:
https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/handle_offline_cpu.sh#L33